lib/internal/test_runner/tap_stream.js

```diff
@@ -131,9 +131,15 @@ class TapStream extends Readable {
 
 // In certain places, # and \ need to be escaped as \# and \\.
 function tapEscape(input) {
-  return StringPrototypeReplaceAll(
-    StringPrototypeReplaceAll(input, '\\', '\\\\'), '#', '\\#'
-  );
+  let result = StringPrototypeReplaceAll(input, '\\', '\\\\');
+  result = StringPrototypeReplaceAll(result, '#', '\\#');
+  result = StringPrototypeReplaceAll(result,'\b', '\\b');
```

Comment:
Same change on the following lines. Other than that, I think this looks good.

```suggestion
  result = StringPrototypeReplaceAll(result, '\b', '\\b');
```

Reply:
Sorry, didn't notice that 😞 
fixed 🚀